python_ta/checkers/function_parameter_not_mentioned_checker.py

```diff
@@ -0,0 +1,56 @@
+"""checker that every function parameter is mentioned by name in the docstring text.
+"""
+
+from __future__ import annotations
+
+import doctest
+import string
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class FunctionParameterNotMentionedChecker(BaseChecker):
```

Comment:
Rename this class, and the file itself, to align with the new name of the check. (Note that this also applies to the test files as well; generally speaking, a "renaming" change can actually affect a lot of places.)